test/manual/amp-base-carousel/multi-visible-single-advance.amp.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html ⚡>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1" />
+    <link rel="preload" as="script" href="https://cdn.ampproject.org/v0.js" />
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+    <script async custom-element="amp-base-carousel" src="https://cdn.ampproject.org/v0/amp-carousel-0.2.js"></script>
```

Comment:
Should this be importing `amp-base-carousel`?

Reply:
Thanks!